src/components/Tipseen/__stories__/tipseen.stories.mdx

```diff
@@ -0,0 +1,228 @@
+        />
+      }
+    >
+      <div className="monday-style-story-tipseen_container" />
+    </Tipseen>
+  </Story>
+</Canvas>
+
+### Tipseen with a wizard
+Use when the tip is too long or when you want to teach something in steps.
+<Canvas>
+  <Story name="Tipseen with a wizard">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <Tipseen
+          modifiers={modifiers}
+          position={Tipseen.positions.RIGHT} 
+          content={
+            <TipseenWizard 
+              title="This is a title"
+              steps={content}
+              activeStepIndex={2}
+            />
+          }
+        >
+          <div className="monday-style-story-tipseen_container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+### Tipseen with image
+<Canvas>
+  <Story name="Tipseen with image">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <Tipseen 
+          position={Tipseen.positions.RIGHT}
+          modifiers={modifiers}
+          isCloseButtonOnImage
+          content={
+            <>
+              <TipseenImage
+                className="monday-style-story-tipseen_image"
+                src={picture}
+              />
+              <TipseenWizard 
+                title="This is a title"
+                steps={content}
+                activeStepIndex={2}
+              />
+            </>
+          }
+        >
+          <div className="monday-style-story-tipseen_big-container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+        <>
+          <div>
+            <div className="monday-style-story-tipseen_column-item">
+              <Icon icon={Board} iconSize="24" /><span>💫 Research tasks</span>
+            </div>
+            <Tipseen
+              modifiers={modifiers}
+              position={Tipseen.positions.RIGHT} 
+              content={
+                <TipseenContent
+                  isDismissHidden={false}
+                  title="Step1 of 7"
+                  isDismissHidden
+                  children="Click on the “Sales” board to get started"
```

Comment:
please pass children content to component using open and closing tags syntax.
for example: <TipseenContent>Click on the “Sales” board to get started</TipseenContent>